src/auslib/services/releases.py

```diff
@@ -18,6 +18,27 @@
 
 store = object()
 
+
+def await_coroutines(coros):
+    # We use `asyncio.run` to wait on the results of a number of coroutines below.
+    # `asyncio.gather` cannot be used directly with it, so we need to wrap it in
+    # a coroutine.
+    async def coroutine_gather(*args, **kwargs):
+        return await asyncio.gather(*args, **kwargs)
+
+    results = asyncio.run(coroutine_gather(*coros, return_exceptions=True))
+    for r in results:
+        if isinstance(r, Exception):
+            # aiohttp exceptions indicate a failure uploading to GCS, which don't warrant
+            # sending an error back to the client.
+            if isinstance(r, ClientError):
```

Comment:
If you want to reuse this function with different constraints, you could add a kwarg tuple that contains the exception types that are logged rather than raised.

Reply:
That's not a bad idea; I think I'll defer it for now since there's no use case for other exceptions at the moment.